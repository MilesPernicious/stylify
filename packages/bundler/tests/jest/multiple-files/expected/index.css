.display\:grid{
	display: grid
}
.transition\:all_\.5s{
	transition: all .5s
}
.color\:red{
	color: red
}
.font-weight\:bold{
	font-weight: bold
}
