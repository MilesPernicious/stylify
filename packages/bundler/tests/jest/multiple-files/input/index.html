<div class="display:grid transition:all_.5s"></div>
