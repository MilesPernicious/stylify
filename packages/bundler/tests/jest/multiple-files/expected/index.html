<div class="display:grid transition:all__.5s"></div>
